### ISSUE TITLE
Gulp seems not to finish task

### ISSUE THREAD

**Issue by nvioli** [PhilJ/gulp-kss#19]
The plugin seems to be working correctly; i.e. the styleguide is generated, but gulp does not seem to report that the task has completed. Here's my taskdef:

    gulp.task('styleguide-main',function(){
    	return gulp.src([paths.metaRoot + 'www/css2015/**/*.less'])
    		.pipe(plugins.kss({
    	        overview:			paths.metaRoot + 'www/css2015/styleguide.md',
    	        templateDirectory:	paths.metaRoot + 'www/css2015/styleguide-template/'
    	    }))
    	    .pipe(gulp.dest(paths.staticOut + 'styleguide/'));
    });

and when I run it on the commandline I only get:

    nvioli-4:node nvioli$ gulp styleguide-main
    [16:46:32] Using gulpfile ~/git/node/gulpfile.js
    [16:46:32] Starting 'styleguide-main'...
    nvioli-4:node nvioli$

All other tasks report "Finished".

Please let me know if I can provide any more information.

**Comment by nvioli** [comment]
Verified it's the same with out the `overview` or `templateDirectory` defined, i.e.

    .pipe(plugins.kss())
has the same result.

**Comment by jamie-pate** [comment]
index.js endStream() function needs
```
            self.emit('end')
```
at the end

**Comment by jamie-pate** [comment]
Looks to be fixed by
https://github.com/PhilJ/gulp-kss/commit/4af6b82a3192ba66b353fce6f591bd2b6d24abc5  ??

**Comment by nvioli** [comment]
Yes I just pulled the latest version of index.js direct from github and this issue is solved.

However this means that the version installed via `npm install` is out of date. Do you need to push a new version over there?

**Comment by demersdesigns** [comment]
@PhilJ Manually pasting in the contents of index.js into the version pulled from NPM works as expected. Would it be possible to get the latest version pushed to NPM?
